test/onnx/pytorch_test_common.py

```diff
@@ -188,3 +193,18 @@ def wrapper(self, *args, **kwargs):
 
 def flatten(x):
     return tuple(function._iter_filter(lambda o: isinstance(o, torch.Tensor))(x))
+
+
+def set_rng_seed(seed):
+    torch.manual_seed(seed)
+    random.seed(seed)
+    np.random.seed(seed)
+
+
+class ExportTestCase(common_utils.TestCase):
```

Comment:
A brief on when it should be used would be helpful!

Reply:
common_utils.TestCase.setUp has some seed setting behaviors iirc. Can we rely on that for torch and np etc.?

Reply:
Good idea, uncovered several flaky tests. Reverting and tracking through https://github.com/pytorch/pytorch/issues/88264

Reply:
I wonder if we could extend this for tests other than runtime cases